cmd/webhook/main.go

```diff
@@ -158,6 +158,9 @@ func newConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 		resolutionv1alpha1GroupVersion = resolutionv1alpha1.SchemeGroupVersion.Version
 		resolutionv1beta1GroupVersion  = resolutionv1beta1.SchemeGroupVersion.Version
 	)
+	// Decorate contexts with the current state of the config.
+	store := defaultconfig.NewStore(logging.FromContext(ctx).Named("config-store"))
```

Comment:
do you know why in the other controllers, `store.WatchConfigs` is called within the inner function?

Reply:
> do you know why in the other controllers, `store.WatchConfigs` is called within the inner function?

Ah, that's a good point, I don't know, but it should be the same for all at least 😛 So I can move it there. It works in both cases, not sure what is the reason.

Reply:
It's because of the difference in the outer function definitions.

Note `newValidationAdmissionController` takes in a parameter of type `string` and returns `func` which has two parameters (`Context`, `Watcher`) and returns `controller.Impl` whereas `newConversionController` takes two parameters (`Context`, `Watcher`) and returns `controller.Impl`.

```
func newValidationAdmissionController(name string) func(context.Context, configmap.Watcher) *controller.Impl { ... }
```

```
func newConversionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl { ... }
```

Does it make sense?